hummingbot/connector/connector_base.pyx

```diff
@@ -14,7 +14,7 @@ from hummingbot.core.event.events import (
 from hummingbot.core.event.event_logger import EventLogger
 from hummingbot.core.network_iterator import NetworkIterator
 from hummingbot.connector.in_flight_order_base import InFlightOrderBase
-from hummingbot.connector.markets_recorder import TradeFill_order_details
+from hummingbot.connector.utils import TradeFill_order_details
```

Comment:
I have to check, but this change would inevitably affect all connectors.

Reply:
This should not be merged, for whatever reason, hummingbot.py would always fail because it couldn't find `TradeFill_order_details`, this is an artefact of my debugging and should have been removed.
Let's track this at https://github.com/CoinAlpha/hummingbot/pull/2823#discussion_r563434135 since I am still having that problem when reverting.